src/Akka.Persistence.MongoDb.Tests/Bug25FixSpec.cs

```diff
@@ -41,6 +41,7 @@ public MyJournalActor(string persistenceId, IActorRef target)
 
             public override string PersistenceId { get; }
         }
+        private static MongoDbConnectionString _mongoDb = new MongoDbConnectionString();
```

Comment:
Should not be `static` - needs to be instanced so there can't be cross-pollination between tests in the same fixture. Same goes for every unit test fixture.

Reply:
Why can't this value just be passed in from the fixture?